CRM/Utils/Mail.php

```diff
@@ -142,7 +142,8 @@ public static function _createMailer($driver, $params) {
 
   /**
    * Wrapper function to send mail in CiviCRM. Hooks are called from this function. The input parameter
-   * is an associateive array which holds the values of field needed to send an email. These are:
+   * is an associateive array which holds the values of field needed to send an email. Note that these
+   * parameters are case specific. The Parameters are:
```

Comment:
I don't want to be too picky but if the purpose is to clarify then it should say "case-sensitive", not "case specific".